design-system-docs/src/_layouts/default.erb

```diff
@@ -1,22 +1,26 @@
 <!doctype html>
 <html lang="<%= site.locale %>">
   <head>
-    <title>Design system docs</title>
+    <meta charset="utf-8" />
+    <%= seo %>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="robots" content="noindex, nofollow">
     <link rel="stylesheet" href="<%= webpack_path "css" %>" />
+    <style><%= Rouge::Theme.find('github').render(scope: '.highlight') %></style>
```

Comment:
Curiosity question - I'm not really sure what the highlight styles do?

Reply:
Good question. It is syntax highlighting for code samples. Bridgetown includes this library https://github.com/rouge-ruby/rouge which has some built in themes. Picked one to get us started, but there's loads available or you can write your own.

Reply:
I see, thanks!